lib/services/nfc.dart

```diff
@@ -132,7 +135,10 @@ class NFCService {
     } on PlatformException {
       print('Method "NFC event started with" exception was thrown');
     }
+    _checkNfcStartedWithTimer?.cancel();
```

Comment:
shouldn't we cancel only when we get a valid payload?

Reply:
The timer is to resolve the delay bug, we need to check for deep link once, because the app is either started with one or not.